automation/nmstate-future-reporter/main.go

```diff
@@ -94,13 +103,20 @@ func generateMessage() (slackMessage, error) {
 	var buildId string
 	var buildStatus finished
 	var jobURL string
+	var vers versions
 	var err error
 
 	if *fakeReport != "" {
 		buildId, buildStatus, jobURL, err = generateFakeData(*fakeReport)
 		if err != nil {
 			return slackMessage{}, fmt.Errorf("failed to generate fake data: %w", err)
 		}
+		vers = versions{
+			Nmstate:               "2.2.55-0.20251031.2666git045ed3c4.el9",
+			NmstateBuildID:        "9752008",
+			NetworkManager:        "1.55.4-34245.copr.5f85b55f7f.el9",
+			NetworkManagerBuildID: "9760340",
```

Comment:
Let's change those to fake. Like `0.0.0-0` or something. So that when you see it, you immediately recognize that you received fake data.

Reply:
I will add the word fake there